returnn/tf/engine.py

```diff
@@ -2338,6 +2343,17 @@ def extra_fetches_callback(seq_idx, seq_tag, **kwargs):
           # Interpret output as bytes/utf8-string.
           outputs[target_idx] = bytearray(outputs[target_idx]).decode("utf8")
 
+      serialized_output = []
+      for target_idx in range(num_targets):
+        if output_layers[target_idx].output.sparse and \
```

Comment:
Do not use `\` at line end. Rather use brackets (`(...)`) when you need to use line breaks.